server/ai_http.go

```diff
@@ -49,6 +50,8 @@ func startAIServer(lp lphttp) error {
 
 	openapi3filter.RegisterBodyDecoder("image/png", openapi3filter.FileBodyDecoder)
 
+	trickle.ConfigureServerWithMux(lp.transRPC)
```

Comment:
Do we maybe want to have a feature flag for this? 

For the context, we'll merge `ai-video` into `master` most probably tomorrow. Then, this change may go to out transcoding broadcaster.

Reply:
This would only be triggered on orchestrators with AI enabled, and in order to do anything, they would also need a live pipeline loaded - otherwise, selection should fail on the gateway side.

We can still add a (temporary?) flag if there is something else that would protect against.

Reply:
Yeah, maybe you're right. We can go with this part without feature flag.